free-1.md

```diff
@@ -57,12 +181,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+For CS111, I think this project would require the most time spent on creating
+an expressive syntax that is liked by lighting designers. The actual translation
+to a protocol that could interact with a dimmer rack is outside the scope of the
+project at this time. However, a "transpiler" that converts the DSL to the
+traditional cue programming langugae is certainly feasible and would be a good
+excersize.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+As mentioned above, this project could have a huge scope if it involved compiling
+to a protocol that is processable by a dimmer rack. In the ultimate project it
+would also involve creating a new lighting console that had a keyboard similar
+to modern day keyboards with shortcuts for common keywords.
 
 ### Benefits and drawbacks
-_Why might this be a good idea for a project? Why might this not be a good idea 
+_Why might this be a good idea for a project? Why might this not be a good idea
 project?_
 
+Getting professional lighting designers and programmers to use it would be
+very difficult since they rely on expensive lighting consoles with custom
+buttons that they are extrmeley familiar with. However, I do think it has
+potential if proven through practice to be easier, more efficient, and descriptive.
+
+Because the potential scope is so large, there will likely only be part of the
+project done by the end of the semester; in an ideal world I would contruct
+the langugae and all the architecture needed to communicate with digital
+dimmer rack, but I do not think this is a realistic goal for the semester.
```

Comment:
I like how realistic you are about the limits of what you might be able to achieve in a semester.